Source/Fuse.Controls.ScrollView/Triggers/WhileVisibleInScrollView.uno

```diff
@@ -120,16 +146,29 @@ namespace Fuse.Triggers
 		
 		void Update()
 		{
-			if (_element == null || _scrollable == null)
+			if (_element == null || _scrollable == null || !_element.HasMarginBox)
 				return;
 				
 			var min = _element.GetLayoutPositionIn(_scrollable);
 			var max = min + _element.ActualSize;
-			
-			var dist = _scrollable.ToScalarPosition(_scrollable.DistanceToView(min, max));
 			var maxDist = _scrollable.ToScalarPosition( RelativeTo.GetPoints(Distance, _scrollable) );
-			
-			SetActive( dist < (maxDist + float.ZeroTolerance) );
+
+			bool isInView = false;
+
+			switch (_how)
+			{
+				case WhileVisibleInScrollViewHow.Full:
+					var distStart = _scrollable.ToScalarPosition(_scrollable.DistanceFromViewStart(min, max));
+					var distEnd = _scrollable.ToScalarPosition(_scrollable.DistanceFromViewEnd(min, max));
+					isInView = (distStart > (maxDist + float.ZeroTolerance)) && (distEnd > (maxDist + float.ZeroTolerance));
```

Comment:
This should probably be `- float.ZeroTolerance`. The comparison is the oppposite as the one below, and the tolerance is supposed to be in the diretion of inclusion, not exclusion.